src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "PRETTY_NAME" | sed 's/PRETTY_NAME=//g' | sed 's/["]//g' | awk '{print $1}'`
```

Comment:
took care of these changes and changes required for other Linux distros as part of: https://github.com/rahulkotecha/corefx/commit/e8acf953de2af1f2fce8f7e98cc6d080d6af42fb
Thanks!